assets/device/classes/airfryer.json

```
@@ -0,0 +1,8 @@
+{
+  "title": {
+    "en": "Airfryer"
+  },
+  "description": {
+    "en": "Use this device class for airfryers."
```

```suggestion
    "en": "Use this device class for air fryers."
```

Seems to be two words in English: https://en.wikipedia.org/wiki/Convection_oven#Air_fryer